src/TTextEdit.cpp

```diff
@@ -993,13 +994,49 @@ void TTextEdit::mouseMoveEvent( QMouseEvent * event )
         }
     }
     x = ( event->x() / mFontWidth ) - timeOffset;
-    if( ( x < 0 ) || ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
+    if( ( y < 0 ) || ( y > (int) mpBuffer->size()-1 ) )
```

Comment:
Just tested by removing that check and dragging a selection above the top of the screen. Yes, it can be < 0. When it is, I get a crash-to-desktop.

If we leave it alone, no drag event will run when you are dragging around above the TextEdit. 
If we slap in `if( y < 0 ) y = 0;` then it will continue processing drag events while the mouse is out of bounds.

Which behavior is preferable?